server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/MailboxFactory.scala

```diff
@@ -112,10 +145,40 @@ class MailboxFactory @Inject() (subscriptionManager: SubscriptionManager) {
         namespace = item.namespace,
         rights = item.rights,
         quotas = item.quotas,
-        isSubscribed = retrieveIsSubscribed(item))
+        isSubscribed = retrieveIsSubscribed(item.mailboxMetaData.getPath, item.mailboxSession))))
+      case item: SingleMailboxItem => SMono.fromCallable(() =>
```

Comment:
(IMO we should consider reactifying this, but maybe as part of another PR/ticket)